pytensor/tensor/math.py

```diff
@@ -733,7 +744,21 @@ def isnan(a):
 
 @scalar_elemwise
 def isinf(a):
-    """isinf(a)"""
+    """
+    isinf(a)
+
+    Elementwise test for infinite value.
+
+    Parameters
+    ----------
+    a : TensorVariable
```

Comment:
All these should have TensorLike (I am not sure the capitalization) as inputs (anything that can be correctly converted to TensorVariables)
```suggestion
    a : TensorLike
```